src/codewhisperer/activation.ts

```diff
@@ -232,6 +239,22 @@ export async function activate(context: ExtContext): Promise<void> {
         )
     }
 
+    function hideStatusBar() {
+        if (isInlineCompletionEnabled()) {
+            InlineCompletionService.instance.hideCodeWhispererStatusBar()
+        } else {
+            InlineCompletion.instance.hideCodeWhispererStatusBar()
+        }
```

Comment:
Are there two status bars now?

Reply:
Only one status bar.

Reply:
Okay, I see now that the goal is to keep the legacy implementation as isolated as possible so it can be cut in its entirety. 

> To avoid the extra cost of maintaining two code paths, in the long run, when most users have migrated to VSC >= 1.68.0, we can drop the previous inline experience by removing onAcceptance.ts and inlineCompletion.ts.

What's our threshold for this? Other than Cloud9, >95% of users are already on 1.68+.

Reply:
>keep the legacy implementation as isolated as possible so it can be cut in its entirety.

Yes.

>What's our threshold for this?

Not sure. This threshold is defined by Product. In my opinion, 95% is close but not good enough. 98% sounds good.